context/app/static/js/components/Detail/provenance/ShowDerivedEntitiesButton/ShowDerivedEntitiesButton.spec.js

```diff
@@ -0,0 +1,8 @@
+// import { render, screen } from 'test-utils/functions';
+import { getUniqueNewSteps } from './ShowDerivedEntitiesButton';
+
+test('should return only unique new steps', () => {
+  const existingSteps = [{ name: 'a' }, { name: 'b' }, { name: 'c' }];
+  const newSteps = [{ name: 'a' }, { name: 'c' }, { name: 'd' }];
+  expect(getUniqueNewSteps(existingSteps, newSteps)).toEqual([{ name: 'd' }]);
+});
```

Comment:
Why not test the component?

Reply:
I could test that it makes the correct calls to the APIs?

Reply:
Hmm, i guess there isn't really much going to test actually.  Feel free to ignore.